src/boot/stylesheets/actions.scss

```diff
@@ -75,34 +75,44 @@
 	&:only-child {
 		padding: 0 18px;
 	}
-
-	a {
-		outline: none;
-	}
-
-	a.inactive-action:hover {
-		color: $orange-jazzy;
-	}
-
-	a.active-action:hover {
-		color: $gray-text-min;
-	}
 }
 
 .wpnc__main div.wpnc__action-link {
+	cursor: pointer;
 	text-align: center;
 	font-size: 12px;
+	height: 55px;
 
 	.wpnc__noticon {
 		font-size: 24px;
 	}
 
-	a.active-action {
+	.active-action {
```

Comment:
The existing modifier classes are fine, but just a heads up that we'll want to namespace our classes with `wpnc` for new class names.